microdaq/macros/microdaq_macros/mdaq_ao_scan_init.sci

```diff
@@ -64,6 +64,7 @@ function  mdaq_ao_scan_init(arg1, arg2, arg3, arg4, arg5, arg6, arg7)
     end
 
     if scan_time < 0 then
```

Comment:
This warning is only to notify the user that he should use -1 instead of value < then 0 - is rather to keep good function call values.